tests/unit/api/main.py

```diff
@@ -1845,26 +1916,32 @@ def test_project_network_bad_access_fails(self):
         for access in ['', 'anvil-oldtimer']:
             for net_id in ['', '35']:
                 with pytest.raises(api.BadArgumentError):
-                    api.network_create('hammernet', 'anvil-nextgen', access, net_id)
+                    api.network_create('hammernet',
+                                       'anvil-nextgen',
+                                       access, net_id)
 
     def test_admin_network(self):
-        """Succesfully create all 4 varieties of administrator-owned networks."""
+        """
+        Succesfully create all 4 varieties of administrator-owned networks.
+        """
         api.project_create('anvil-nextgen')
         project = api._must_find(model.Project, 'anvil-nextgen')
-        for project_api, project_db in [('', None), ('anvil-nextgen', project)]:
+        for project_api, project_db in [('', None), ('anvil-nextgen', project)]:  # noqa
             for net_id, allocated in [('', True), ('35', False)]:
                 network = 'hammernet' + project_api + net_id
                 api.network_create(network, 'admin', project_api, net_id)
                 network = api._must_find(model.Network, network)
                 assert network.creator is None
                 assert network.access is project_db
                 assert network.allocated is allocated
-            network = api._must_find(model.Network, 'hammernet' + project_api + '35')
+            network = api._must_find(model.Network, 'hammernet' + project_api + '35')  # noqa
```

Comment:
This should look reasonable split across two lines. There's got to be a way to deal with the for loop as well.